lib/rules/block-opening-brace-space-before/__tests__/index.js

```diff
@@ -80,89 +80,103 @@ testRule(rule, {
 	],
 });
 
-testRule(rule, {
-	ruleName,
-	config: ['always', { ignoreAtRules: ['for'] }],
-	fix: true,
+for (const selector of ['for', '/fo/', /fo/]) {
```

Comment:
Thanks for cleaning up the tests!

I suggest we remove the `for` loop and place the array within the config so that it is idomatic with our other test files, like so:

```
config: ['always', { ignoreAtRules: ['for', '/fo/', /fo/] }],
```